api_docs/writing-bots.md

```diff
@@ -17,24 +17,28 @@ On this page you'll find:
 
 ## Installing a development version of the Zulip bots package
 
+{start_tabs}
+
 1. `git clone https://github.com/zulip/python-zulip-api.git` - clone the [python-zulip-api](
   https://github.com/zulip/python-zulip-api) repository.
 
-2. `cd python-zulip-api` - navigate into your cloned repository.
+1. `cd python-zulip-api` - navigate into your cloned repository.
 
-3. `python3 ./tools/provision` - install all requirements in a Python virtualenv.
+1. `python3 ./tools/provision` - install all requirements in a Python virtualenv.
 
-4. The output of `provision` will end with a command of the form `source .../activate`;
+1. The output of `provision` will end with a command of the form `source .../activate`;
    run that command to enter the new virtualenv.
 
-5. *Finished*. You should now see the name of your venv preceding your prompt,
+1. *Finished*. You should now see the name of your venv preceding your prompt,
    e.g. `(zulip-api-py3-venv)`.
 
 *Hint: `provision` installs the `zulip`, `zulip_bots`, and
```

Comment:
Shouldn't this be formatted in our tip markdown formatting? Instead of being in the instruction tabs block.